src/backend.py

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env python
+# -*-coding:utf-8-*-
+
```

Comment:
this is useless and even missleading as Galaxy uses its own python.

Reply:
removed.